<a href="https://colab.research.google.com/github/rishika-p/assignment-1/blob/master/q7.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

In [2]:
n=int(input("enter number between 100 and 1000:"))
rem=0
sum=0
if n>=100 and n<=1000:
  while n>0:
    rem=n%10
    sum=sum+rem
    n=n//10
  print(sum)
else:
  print("please enter number between given range")  


enter number between 100 and 1000:745
16


WRITE A PROGRAM THAT READS AN INTEGER BETWEEN 100 AND 1000 AND ADD ALL THE DIGITS IN THE INTEGER (ex.INPUT 745 #OUTPUT=16(7+4+5))